examples/example01-car-trade/validatorDriver/src/app.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2019-2020 Fujitsu Laboratories Ltd.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * app.ts
+ */
+
+/* Summary:
+ *
+ */
+
+var express = require('express');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+var fabricAccess = require('./fabricValidatorAccess');
+var ethereumAccess = require('./ethereumValidatorAccess');
+
+var app = express();
+
+app.use(bodyParser.json());
+//app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+
+app.get('/validatorDriver', function (req, res, next) {
+    console.log('call GET : /validatorDriver');
+    var validator = req.query.validator;
+    var func = req.query.func;
+    var param = req.query.param;
+    console.log('validator : ' + validator);
+    console.log('func : ' + func);
+    console.log('param : ' + param);
+
+    xxx(validator, func, param, res, next);
+});
+
+
+app.post('/validatorDriver', function (req, res, next) {
+    console.log('call POST : /validatorDriver');
+    var validator = req.body.validator;
+    var func = req.body.func;
+    var param = req.body.param;
+    console.log('req.body : ' + JSON.stringify(req.body));
+    console.log('validator : ' + validator);
+    console.log('func : ' + func);
+    console.log('param : ' + param);
+
+    xxx(validator, func, param, res, next);
+});
+
+function xxx(validator, func, param, res, next): void {
```

Comment:
I personally don't have anything against having a function called xxx, but you might want to avoid it anyway because of this alternative meaning explained here: https://en.wikipedia.org/wiki/.xxx
@takeutak What do you think?

Reply:
Whoops. The cultural gap is here. 
We should be used more safe strings like foo or bar > @takeutak